Clarify about why GitHub Discussions is disabled (#43)
quarkiverse/quarkiverse-wiki: docs/Home.md

Extensions generated with the `create-extension` goal of the Quarkus maven plugin expect code to be formatted according to the Quarkus conventions as documented in: https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#ide-config-and-code-style. The default CI setup will validate the format and fail your build if the files don't conform.

## Discussions 

> Because we want to centralize discussions in common places so developers including core committers can contribute, GitHub Discussions are disabled in Quarkiverse extension repositories. Development questions should be addressed in the [Mailing List](https://groups.google.com/g/quarkus-dev) and user questions in [Stack Overflow](https://stackoverflow.com/questions/tagged/quarkus).

Reviewer: ```suggestion
To avoid spreading information too much and make it hard to figure out where to post we have following default guidelines:

Development questions should be addressed in the [Mailing List](https://groups.google.com/g/quarkus-dev) and user questions in [Stack Overflow](https://stackoverflow.com/questions/tagged/quarkus) + the quarkiverse project tag.
```

> However we can enable it in the repository if the extension maintainer really wants it (open an issue requesting that and ping the `@quarkiverse/quarkiverse-owners` team in the issue). However be aware that Core developers will only be aware of these discussions only if they are explicitly mentioned.

Reviewer 2: I would rather be far more neutral.

> GitHub Discussions are disabled by default in Quarkiverse extension repositories.
>
> If you wish to enable them for the internal discussions about your project, please create an issue (with a link to the appropriate issue tracker) so that we enable it for you.

We could even have a template so people just have to fill in the repository and that's it.

Author: That depends on which repository you are opening the issue. I was expecting to come from the extension's repository, so you don't need to fill anything

> However we can enable it in the repository if the extension maintainer really wants it (open an issue requesting that and ping the `@quarkiverse/quarkiverse-owners` team in the issue). However be aware that Core developers will only be aware of these discussions only if they are explicitly mentioned.

Reviewer: ```suggestion
However we can enable Discussions feature in the repository if the extension maintainer really wants it (open an issue requesting that and ping the `@quarkiverse/quarkiverse-owners` team in the issue). However be aware that Core developers will not follow or be aware of these discussions unless explicitly mentioned.
```
